src/Data/Foreign/NullOrUndefined.purs

```diff
@@ -28,3 +28,5 @@ unNullOrUndefined (NullOrUndefined m) = m
 readNullOrUndefined :: forall a. (Foreign -> F a) -> Foreign -> F (NullOrUndefined a)
 readNullOrUndefined _ value | isNull value || isUndefined value = pure (NullOrUndefined Nothing)
 readNullOrUndefined f value = NullOrUndefined <<< Just <$> f value
+
+foreign import writeNull :: Foreign
```

Comment:
tests:

```purescript
testRoundTrip [NullOrUndefined (Just "test")]
testRoundTrip [NullOrUndefined (Nothing :: Maybe String)]
```

output:

```purescript
["test"]
[null]
```

looks kinda janky... maybe something i need to look into from purescript-foreign?